### PR TITLE
[Redesign]Update Manage Packages to be more compact.

### PR DESCRIPTION
**Pull request by ryuyu**
Make manage package page more compact through horizontal layout.

First part of addressing https://github.com/NuGet/NuGetGallery/issues/4431

@joelverhagen @jonwchu 

Preview:
image

### CODE REVIEW

File: src/NuGetGallery/Views/Users/_UserPackagesList.cshtml [hunk 1]
```diff
@@ -21,10 +21,53 @@
...
+                                     @ViewHelpers.ImageFallback(Url.Absolute("~/Content/gallery/img/default-package-icon-256x256.png")) />
+                            </td>
+                            <td class="col-sm-3 align-middle"><a href="@Url.Package(package.Id)">@package.Id</a></td>
+                            <td class="col-sm-2 align-middle">
+                            @foreach (var owner in package.Owners)
+                            {
+                                <a href="@Url.User(owner.Username)">@owner.Username  </a>
+                            }
+                            </td>
+                            <td class="col-sm-2 align-middle">@package.DownloadCount</td>
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/_UserPackagesList.cshtml]: Spoke offline, this should be the total download count across all versions

File: src/NuGetGallery/Views/Users/_UserPackagesList.cshtml [hunk 2]
```diff
@@ -21,10 +21,53 @@
...
+                            <th class="col-sm-1"></th>
+                            <th class="col-sm-3">Package Id</th>
+                            <th class="col-sm-2">Authors</th>
+                            <th class="col-sm-2">Downloads</th>
+                            <th class="col-sm-2">Latest Version</th>
+                            <th class="col-sm-2"></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                    @foreach (var package in @Model.Packages)
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/_UserPackagesList.cshtml]: This should be sorted by ID (probably on the controller, not in the view)

Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/_UserPackagesList.cshtml]: (also spoke offline)

File: src/NuGetGallery/Views/Users/_UserPackagesList.cshtml [hunk 3]
```diff
@@ -21,10 +21,53 @@
...
+                    {
+                        //@Html.Partial("_ListPackage", package)
+                        <tr class="manage-package-listing">
+                            <td class="col-sm-1 align-middle">
+                                <img class="package-icon img-responsive" aria-hidden="true" alt="" 
+                                     src="@(PackageHelper.ShouldRenderUrl(package.IconUrl) ? package.IconUrl : Url.Absolute("~/Content/gallery/img/default-package-icon.svg"))"
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/_UserPackagesList.cshtml]: This should have onerror for machines that do not support SVG

File: src/NuGetGallery/Views/Users/_UserPackagesList.cshtml [hunk 4]
```diff
@@ -21,10 +21,53 @@
...
+                            {
+                                <a href="@Url.User(owner.Username)">@owner.Username  </a>
+                            }
+                            </td>
+                            <td class="col-sm-2 align-middle">@package.DownloadCount</td>
+                            <td class="col-sm-2 align-middle">@package.Version</td>
+                            <td class="col-sm-2 text-right align-middle">
+                                <div class="btn-group">
+                                    <a href="@Url.EditPackage(package.Id, package.Version)" class="btn" title="Edit Package">
+                                        <i class="ms-Icon ms-Icon--Edit" aria-hidden="true"></i>
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/_UserPackagesList.cshtml]: Is this accessible? There is no text node and the icon is `aria-hidden`.

File: src/NuGetGallery/Views/Users/_UserPackagesList.cshtml [hunk 5]
```diff
@@ -21,10 +21,53 @@
...
+                            <td class="col-sm-1 align-middle">
+                                <img class="package-icon img-responsive" aria-hidden="true" alt="" 
+                                     src="@(PackageHelper.ShouldRenderUrl(package.IconUrl) ? package.IconUrl : Url.Absolute("~/Content/gallery/img/default-package-icon.svg"))" 
+                                     @ViewHelpers.ImageFallback(Url.Absolute("~/Content/gallery/img/default-package-icon-256x256.png")) />
+                            </td>
+                            <td class="col-sm-3 align-middle"><a href="@Url.Package(package.Id)">@package.Id</a></td>
+                            <td class="col-sm-2 align-middle">
+                            @foreach (var owner in package.Owners)
+                            {
+                                <a href="@Url.User(owner.Username)">@owner.Username  </a>
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/_UserPackagesList.cshtml]: what happens when there are so many owners that it takes up more than the width of the column? Does it stretch? or wrap to second line?

Comment (ryuyu) [inline src/NuGetGallery/Views/Users/_UserPackagesList.cshtml]: It wraps to a second line.

**Pull request merged**